crates/holochain/Cargo.toml

```diff
@@ -268,6 +268,14 @@ sqlite = [
 # Extremely verbose wasm memory read/write logging
 wasmer_debug_memory = ["holochain_wasmer_host/debug_memory"]
 
+# Enable wasm compiler
+# Incompatible with "wasmer_wamr"
+wasmer_sys = ["wasmer-middlewares", "wasmer/default", "holochain_wasmer_host/wasmer_sys"]
+
+# Enable wasm interpreter
+# Incompatible with "wasmer_sys"
+wasmer_wamr = ["wasmer/wamr", "holochain_wasmer_host/wasmer_wamr"]
```

Comment:
These aren't the most clear feature flag names (they mirror wasmer's feature flag names), should I change them to something like wasmer_compiler & wasmer_interpreter?